<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::QuickReportBrowser</class>
 <widget class="QWidget" name="tt3::gui::QuickReportBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="controlsPanel" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="showLabel">
        <property name="text">
         <string>Show:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="quickReportComboBox"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="refreshLabel">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="refreshIntervalComboBox"/>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="refreshPushButton">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/RefreshSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/RefreshSmall.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="quickReportPanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quickReportComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>tt3::gui::QuickReportBrowser</receiver>
   <slot>_quickReportComboBoxCurrentIndexhanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshIntervalComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>tt3::gui::QuickReportBrowser</receiver>
   <slot>_refreshIntervalComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::QuickReportBrowser</receiver>
   <slot>_refreshPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_quickReportComboBoxCurrentIndexhanged(int)</slot>
  <slot>_refreshIntervalComboBoxCurrentIndexChanged(int)</slot>
  <slot>_refreshPushButtonClicked()</slot>
 </slots>
</ui>
